docs/guide/deploying_contracts.rst

```diff
@@ -75,9 +68,7 @@ Cairo1 contracts
 Declaring Cairo1 contracts
 ##########################
 
-| Starknet 0.11 introduced the ability to declare contracts written in Cairo1!
-
-To declare a new contract, Declare v2 or Declare v3 transaction has to be sent.
+To declare a contract in Cairo version 1 or higher, Declare V2 or Declare V3 transaction has to be sent.
```

Comment:
Nit: maybe also add a similar note above for Cairo 0?

Reply:
Added info about it under `Declaring contracts` section.